lib/charms/glauth_k8s/v0/ldap.py

```diff
@@ -203,6 +204,8 @@ def load(
     ) -> "Secret":
         try:
             secret = charm.model.get_secret(label=label)
+            if content:
+                secret.set_content(content=content)
```

Comment:
Have a question here: why does it need to update the content?

From what I understand, the `load` factory method either gets the existing secret or creates a new one if nothing is found.